app/services/vot/component_value.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
 module Vot
-  ComponentValue = Data.define(:name, :description, :implied_component_values, :requirements).freeze
+  class ComponentValue < Data.define(:name, :description, :implied_component_values, :requirements)
```

Comment:
`Data.define` was designed for struct/class-like objects rather than pure classes. i think if we're going to make this into a true class, we should use an initializer, so my preference for now would be reverting this back for now to the original implementation

Reply:
That's fair. From an inspection and coding support standpoint, the class definition is easier to work with, but I'll revert the change.

Reply:
Fixed.